src/test/java/hw2/ex2/SecondExerciseTest.java

```diff
@@ -0,0 +1,166 @@
+        softAssert.assertTrue(radioSelen.isDisplayed());
+        softAssert.assertAll();
+
+        // 9. Assert that there is Right Section
+        assertTrue(driver.findElement(By.id("mCSB_2")).isDisplayed());
+
+        // 10. Assert that there is Left Section
+        assertTrue(driver.findElement(By.id("mCSB_1")).isDisplayed());
+
+        // 11. Select checkboxes
+        // 12. Assert that for each checkbox there is an individual log
+        // row and value is corresponded to the status of checkbox.
+        checkboxWater.click();
+        WebElement trueLogCheckboxWater = driver.findElement(By.
+                xpath("//ul[@class='panel-body-list logs']//li[contains(.,'Water')]"));
+        assertTrue(trueLogCheckboxWater.getText().contains("Water: condition changed to true"));
+        checkboxWind.click();
+        WebElement trueLogCheckboxWind = driver.findElement(By.
+                xpath("//ul[@class='panel-body-list logs']//li[contains(.,'Wind')]"));
+        assertTrue(trueLogCheckboxWind.getText().contains("Wind: condition changed to true"));
+
+        // 13. Select radio
+        // 14. Assert that for radiobutton there is a log row
+        // and value is corresponded to the status of radiobutton.
+        radioSelen.click();
+        WebElement trueLogradioSelen = driver.findElement(By.
+                xpath("//ul[@class='panel-body-list logs']//li[contains(.,'metal')]"));
+        assertTrue(trueLogradioSelen.getText().contains("value changed to Selen"));
+
+        // 15. Select in dropdown
+        // 16. Assert that for dropdown there is a log row
+        // and value is corresponded to the selected value.
+        Select colorsSelector = new Select(driver.findElement(By
+                .xpath("//div[@class='colors']//select[@class='uui-form-element']")));
+        colorsSelector.selectByVisibleText("Yellow");
+        WebElement trueLogColorsSelector = driver.findElement(By.
+                xpath("//ul[@class='panel-body-list logs']//li[contains(.,'Colors')]"));
+        assertTrue(trueLogColorsSelector.getText().contains("Colors: value changed to Yellow"));
+
+        // 17. Unselect and assert checkboxes
+        // 18. Assert that for each checkbox there is an individual log row
+        // and value is corresponded to the status of checkbox.
+        checkboxWater.click();
+        WebElement falseLogCheckboxWater = driver.findElement(By.
+                xpath("//ul[@class='panel-body-list logs']//li[contains(.,'Water')]"));
+        assertTrue(falseLogCheckboxWater.getText().contains("Water: condition changed to false"));
+        checkboxWind.click();
+        WebElement falseLogCheckboxWind = driver.findElement(By.
+                xpath("//ul[@class='panel-body-list logs']//li[contains(.,'Wind')]"));
+        assertTrue(falseLogCheckboxWind.getText().contains("Wind: condition changed to false"));
+
```

Comment:
I think you missed one important assert here for the `soft assertions`

Reply:
fixed